pkg/api/jobservice/jobservice.proto

```diff
@@ -0,0 +1,22 @@
+syntax = 'proto3';
+
+package jobservice;
+
+message JobServiceRequest {
+    string job_id = 1;
+    string job_set_id = 2;
+    string queue = 3;
+}
+
+message JobServiceResponse {
```

Comment:
(this is expected as events come through async, so it's quite possible you'll have no status when you first ask).

I guess the options here are to have a special status of "UNKOWN", or else return a `protobuf.empty`

Reply:
That's good but how do you error if job id is bad?  If someone calls with a job id that won't ever exist?

Reply:
I don't think we can ever handle this situation due to the way Armada works. Essentially you can't really distinguish between "this is a non existent job id" and "this is a valid job id, but we just don't have any events yet".
For the Airflow use case I don't really think this is a problem.  Essentially the operator is going to:

1. Submit a job to Armada.  This call returns a JobId if Armada accepts the job and an error otherwise.
2. Poll the job service for the status of that job.

In this case, the operator will only be polling if it got a success (and a JobId) from Armada.  If this is the case, Armada guarantees that you'll get messages about the job (even if it's just a message to say that we rejected the job because it was somehow invalid).